source/includes/_isaadmin.md

```diff
@@ -374,11 +374,13 @@ In the event of a death of an Investor, specific tasks need to be carried out. T
 ### 1) The investor's cash and/or investments are being transferred to a surviving spouse on the same platform using an Additional Permitted Subscription ('APS')
 
 If you want to transfer investments to a surviving spouse, you must make sure you have the appropriate legal paperwork and technical capability in place to transfer investments on your platform.  Alternatively, you can wait for investments to mature and only transfer cash when it becomes available.
+
 The process flow is:
+
 - Notify the Goji operations team of the death by emailing platformsupport@goji.investments.
-- Goji will create a 'Death of investor' workflow and lock the account.
+- Goji will create a 'Death of investor' workflow and lock the account. 
```

Comment:
oh, is it just a space at the end here and removal of a space between sentences below?

Reply:
if that's it then, yeah, sure. lgtm

Reply:
Yeah, that's it just a cleanup. Thanks!